adapters/alpies-bsc/index.js

```diff
@@ -10,13 +10,13 @@ const path = require("path");
 
 class AlpiesBSC {
     constructor() {
-        this.name = "alpies-bsc";
-        this.symbol = null;
+        this.name = "Alpies";
+        this.symbol = "ALPIES";
         this.token = "0x0000000000000000000000000000000000000000";
         this.protocol = "binance-smart-chain";
         this.block = 11873270;
-        this.contract = "0x57A7c5d10c3F87f5617Ac1C60DA60082E44D539e";
-        this.events = ["NameRegistered", "NameRenewed"];
+        this.contract = "0x57a7c5d10c3f87f5617ac1c60da60082e44d539e";
+        this.events = ["LogMintReserve", "LogPreMint", "LogMint", "LogSetBaseURI", "LogReveal", "LogRefund"];
```

Comment:
Same issue here: https://github.com/dappradar/nft-sales-adapters/pull/4/files/003ecb0a3267d61fbf28c3cecbfc4d68180eb38c#r823431288

Personally I think that these events are useless for tracking sales:
* `LogSetBaseURI`
* `LogRefund`
* (there might be more, but these are obvious)

Reply:
@liutaurasjanuskas I fixed according to your comments. I already request for a new API Key since the old one was expired, but haven't got a new one.